.github/workflows/modern-ci.yml

```
@@ -0,0 +1,202 @@
+      with:
+        name: dusk-screenshots
+        path: tests/Browser/screenshots/
+        retention-days: 3
+
+    - name: Upload Logs on Failure
+      if: failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: laravel-logs
+        path: storage/logs/
+        retention-days: 3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)